addons/mail/static/src/models/message/message.js

```diff
@@ -544,6 +544,20 @@ function factory(dependencies) {
             return parseAndTransform(prettyBody, addLink);
         }
 
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeMessageTypePrefix() {
+            if (this.message_type === 'notification') {
+                return this.env._t("System notification by ");
+            }
+            if (!this.is_discussion && !this.is_notification) {
+                return this.env._t("Note by ");
+            }
+            return clear();
```

Comment:
Why is there no "Message by" in the standard case?

Reply:
Because I was following the same logic as in v12 (when this text was visible), where message was assumed. But you're right, done, better implicit than explicit.